platform/viewer/public/config/dicomweb_relative.js

```diff
@@ -0,0 +1,160 @@
+window.config = {
```

Comment:
This is the old local_static.js configuration, needed for any site that uses a single domain for both dicomweb and OHIF, but is otherwise just a copy.